File: Peaker.TimeManagment/App/src/Models/EntryFilter.ts
export class EntryFilter {
    CurrentUserDetailId: number;
    UserDetailId?: number;
    ShowAllUsers: boolean;
    EntryId?: number;
    FilterStartDate?: Date;
    FilterEndDate?: Date;
    ExportedToNavision?: boolean;
    RequireJobCode?: boolean;
    ExportedToPayroll?: boolean;

    constructor(currentUserId: number, userId?: number, showAllUsers?: boolean, entryId?: number, startDate?: Date, endDate?: Date, exportedToNavision?: boolean, requireJobCode?: boolean, exportedToPayroll?: boolean) {
        this.CurrentUserDetailId = currentUserId;
        this.UserDetailId = userId;
        this.ShowAllUsers = showAllUsers;
        this.EntryId = entryId;
        this.FilterStartDate = startDate;
        this.FilterEndDate = endDate;
        this.ExportedToNavision = exportedToNavision;
        this.RequireJobCode = requireJobCode;
        this.ExportedToPayroll = exportedToPayroll;
    }
}

File: Peaker.TimeManagment/Models/Filters/EntryFilter.cs
using System;

namespace Peaker.TimeManagment.Models.Filters
{
    public class EntryFilter
    {
        public int CurrentUserDetailId { get; set; }
        public int? UserDetailId { get; set; }
        public bool ShowAllUsers { get; set; }
        public int? EntryId { get; set; }
        public DateTime? FilterStartDate { get; set; }
        public DateTime? FilterEndDate { get; set; }
        public bool? Exported { get; set; }
    }
}